server/util/util.go

```diff
@@ -107,3 +114,34 @@ func Deduplicate(a []string) []string {
 
 	return result
 }
+
+func GetBodyForExcerpt(htmlBodyValue string) string {
```

Comment:
I guess we are using this to remove HTML from the body and just show up the text?

Reply:
Yes